CMakeLists.txt

```
@@ -75,12 +75,16 @@ else()
     # Link to CMake libusb
     target_link_libraries(${TARGET_NAME} PRIVATE usb-1.0)
     if(WIN32 AND NOT MINGW)
-        target_link_libraries(${TARGET_NAME} PRIVATE delayimp.lib Pathcch.lib)
+        target_link_libraries(${TARGET_NAME} PRIVATE delayimp.lib)
         # workaround https://gitlab.kitware.com/cmake/cmake/-/issues/20022
         target_link_options(${TARGET_NAME} PUBLIC "$<LINK_ONLY:/DELAYLOAD:libusb-1.0$<$<CONFIG:Debug>:d>.dll>")
     endif()
 endif()
 
+if(WIN32 AND NOT MINGW)
+    target_link_libraries(${TARGET_NAME} Pathcch.lib)
```

```suggestion
    target_link_libraries(${TARGET_NAME} PRIVATE Pathcch.lib)
```

Shouldn't this remain `PRIVATE` as before? (not sure if that is the default)

---

Sure, nice catch! `PRIVATE` is the default but you can't mix plain and keyword signatures (see https://cmake.org/cmake/help/latest/policy/CMP0023.html), so adding `PRIVATE` is totally the correct thing to do.